src/Glpi/Form/ConditionalVisiblity/ConditionData.php

```diff
@@ -59,9 +59,9 @@ public function getItemUuid(): string
         return $this->item_uuid;
     }
 
-    public function getItemType(): string
+    public function getItemType(): Type
     {
-        return $this->item_type;
+        return Type::tryFrom($this->item_type ?? "") ?? Type::QUESTION;
```

Comment:
Is there any valid use case where it can happen to have a `null` value in `$this->item_type` or to have a value that does not match the `Type` values ?
Maybe an exception would mbe more appropriate here, rather than a fallback to `Type::QUESTION`.

Reply:
I agree.